.travis.yml

```diff
@@ -19,8 +19,7 @@ services:
 #   - docker_images
 
 before_install:
-- composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts
-- sudo service mysql stop
+- docker load -i docker_images/images.tar || true
```

Comment:
I think this should enable caching in combination with this lines:

```
# cache:
#   directories:
#   - docker_images

before_install:
- docker load -i docker_images/images.tar || true

# before_cache:
# - docker save -o docker_images/images.tar $(docker images -a -q)
```
Any chance you tested it? @itaymelamed and I tried to use this to improve the build performance with no luck so far, but it seems your latests build with multi-stage support is now 3 minutes faster.